kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -709,31 +725,111 @@ public Properties getOriginalProperties() {
     return properties;
   }
 
+  public final Map<String, Object> getSchemaRegistryConfigs() {
+    HashMap<String, Object> possibleConfigs = new HashMap<>();
+
+    // Use a default schema.registry.url, if not specified.
+    possibleConfigs.put(SCHEMA_REGISTRY_URL_CONFIG, SCHEMA_REGISTRY_URL_DEFAULT);
```

Comment:
i don't think it's a good idea to have a default for this, unless that is replicating existing behavior. if that's the case, consider depreciating for v3.

in particular, the presence of this property is what i'd use to determine if you instantiate a schema registry instance or not, i think. i guess i'll see how you do that.

Reply:
We actually don't need it in my new implementation, but the old implementation needed it, so now we are stuck on having this default value for `schema.registry.url` because there might be people out there relying on it.